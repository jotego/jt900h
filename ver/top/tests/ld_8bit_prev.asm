    maxmode on
    relaxed on
    org 0
    ; check register in previous bank
    ld a',0xe0
    ld w',0xe1
    ld c',0xe4
    ld b',0xe5
    ld e',0xe8
    ld d',0xe9
    ld l',0xec
    ld h',0xed

    ld ixl,0xf0
    ld ixh,0xf1
    ld iyl,0xf4
    ld iyh,0xf5
    ld izl,0xf8
    ld izh,0xf9
    ld spl,0xfc
    ld sph,0xfd
    nop
    ld (0xffff),0xff
    end