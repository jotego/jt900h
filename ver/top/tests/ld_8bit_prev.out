00000000 - 00000000 - 00000000 - 0000e1e0
00000000 - 00000000 - 00000000 - 0000e5e4
00000000 - 00000000 - 00000000 - 0000e9e8
00000000 - 00000000 - 00000000 - 0000edec
-- index registers --
0000f1f0 - 0000f5f4 - 0000f9f8 - 0000fdfc
SR = 00f8
